tests/test_watson.py

```diff
@@ -737,3 +737,39 @@ def test_report(watson):
     assert report['projects'][0]['name'] == 'foo'
     assert len(report['projects'][0]['tags']) == 1
     assert report['projects'][0]['tags'][0]['name'] == 'B'
+
+
+# renaming project updates frame last_updated time
+def test_rename_project_with_time(mock, watson):
+    """
+    Renaming a project should update any the "last_updated" time on any
```

Comment:
update any the "last_updated"

--->

    update the "last_updated"

Reply:
done